util/oteltest/pyproject.toml

```
@@ -0,0 +1,51 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "oteltest"
+dynamic = ["version"]
+description = "OpenTelemetry Tester"
+readme = "README.md"
+requires-python = ">=3.8"
+license = "Apache-2.0"
+keywords = []
+authors = [
+  { name = "OpenTelemetry Authors", email = "cncf-opentelemetry-contributors@lists.cncf.io" },
+]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Typing :: Typed",
+]
+dependencies = [
+  "opentelemetry-proto ~= 1.24.0",
+  "opentelemetry-api ~= 1.24.0",
+  "grpcio",
+  "protobuf",
+]
+
+[project.urls]
+Documentation = "https://github.com/open-telemetry/opentelemetry-python-contrib#readme"
+Issues = "https://github.com/open-telemetry/opentelemetry-python-contrib/issues"
+Source = "https://github.com/open-telemetry/opentelemetry-python-contrib/"
```

Yay we've found a home for this!

Maybe we could have more specific URLs:
```suggestion
Documentation = "https://github.com/open-telemetry/opentelemetry-python-contrib/util/oteltest#readme"
Issues = "https://github.com/open-telemetry/opentelemetry-python-contrib/issues"
Source = "https://github.com/open-telemetry/opentelemetry-python-contrib/util/oteltest"
```

---

Thanks Tammy 🙏 

I've updated the URLs (I think we'd have to wait until this PR lands before we can test them).